### PR TITLE
chore(upgrade): Ruby 2.7 to Ruby 3 

### PR DESCRIPTION
**Pull request by jpotts244**
Get Doppler running on Ruby 3.0.x!
and removes some custom config that was needed to patch us when using Ruby 2.7

## Fun facts
- Retire `webrick` usage
- Updates version of bundler used in the project

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ gem "sentry-rails"
 gem "twitter-bootstrap-rails"
 gem "twitter-bootstrap-rails-confirm"
 gem "uglifier"
+gem "webrick"
```
Comment (jonallured) [inline Gemfile:33]: lol wut

Comment (jpotts244) [reply Gemfile:33]: oooo you know what, look at this: https://github.com/artsy/doppler/blob/6a13a1cd7fa52026fe5c983be2134c26cfd52282/spec/support/capybara.rb#L5

Comment (jpotts244) [reply Gemfile:33]: lets kill that and then remove the need for `webrick` altogether!

Comment (jpotts244) [reply Gemfile]: Also found some newrelic config in here! Removing

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**